index.md

```diff
@@ -48,11 +30,11 @@ We're located on the 11th floor. Park in the garage and take the elevator to the
 Email: <info@paircolumbus.org>
 
 ### Friends & Links
-- @paircolumbus
+- [GitHub/paircolumbus](https://github.com/paircolumbus)
```

Comment:
Ah, I see. I'm starting to agree. Do you want me to remove the mention plugin and use the old GitHub links or configure it to use Twitter instead of GitHub?

Good point, I forgot that was copied between challenges. Don't forget that you can link to specific headings by id.